packages/flutter/lib/src/gestures/events.dart

```diff
@@ -636,6 +643,7 @@ class PointerCancelEvent extends PointerEvent {
     buttons: buttons,
     down: false,
     obscured: obscured,
+    pressure: pressure,
     pressureMin: pressureMin,
     pressureMax: pressureMax,
     distance: distance,
```

Comment:
i don't understand any of the changes in this file.

Reply:
Ok, let's talk about it tomorrow!